README.md

```diff
@@ -327,9 +327,7 @@ type modules =
         | "dashesOnly"
         | ((name: string) => string);
       exportOnlyLocals: boolean;
-      getJSON:
-        | string
-        | ((resourcePath: string, json: object, outputPath: string) => any);
```

Comment:
I removed `outputPath` as an argument here since that was an assumption made with `getJSON: true`.  We shouldn't assume what the output path _should_ be so we can just provide the user with actually relevant information.